lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2120,6 +2132,15 @@ protected static function compute_style_properties( $styles, $settings = array()
 				}
 			}
 
+			// Processes background styles.
+			if ( $value_path[0] === 'background' && isset( $styles['background'] ) ) {
```

Comment:
This block is here because it's before the `is_array( $value )` check below.

We want to support `"backgroundImage"  <string|array>"` types.